main.py

```diff
@@ -244,26 +247,21 @@ async def fill_blacklist(self) -> None:
         records = await self.pool_pg.fetch("SELECT snowflake_id FROM blacklist")
         self.blacklist = {r["snowflake_id"] for r in records}
 
-    async def get_prefix(self, message: discord.Message) -> Union[List[str], str]:
+    async def _prefix_for_message(self, message: discord.Message) -> str:
```

Comment:
why does it need to be a separate method

Reply:
because they do different things